src/map-from-async.ts

```diff
@@ -62,4 +63,17 @@ export const mapFromAsync = {
   undefined(): Promise<undefined> {
     return Promise.resolve(undefined);
   },
+
+  /**
+   * Maps the specified properties to themselves as is
+   *
+   * ```ts
+   * const objectMapperSchema = {
+   *   ...mapFrom.pick('a', 'b', 'c', 'd'),
+   * }
+   * ```
+   */
+  pick<TKey extends PropertyKey>(...keys: TKey[]) {
```

Comment:
🙏 Since this doesn't do anything different from the sync version, I think we can leave this out of `mapFromAsync`. Users can just import it from `mapFrom`.

Reply:
☑️ Removed `pick` method from `mapFromAsync` 👌